src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -66,6 +66,20 @@
             -n domain-name
 """
 
+helps['cdn custom-domain enable-https'] = """
+type: command
+short-summary: Enable HTTPS delivery of the custom domain.
+examples:
+  - name: Enable HTTPS using a CDN-managed certificate
+    text: >
+        az cdn custom-domain enable-https -g group --profile-name profile --endpoint-name endpoint \\
+            -n domain-name
+  - name: Enable HTTPS using a CDN-managed certificate and a minimum TLS version of 1.2
+    text: >
+        az cdn custom-domain enable-https -g group --profile-name profile --endpoint-name endpoint \\
+            -n domain-name --min-tls-version=1.2
```

Comment:
Can we use --min-tls-version 1.2 to keep consistent with other arguments?

Reply:
Fixed.